aruna/api/storage/services/v2/info_service.proto

```diff
@@ -132,9 +143,14 @@ message GetPubkeysResponse {
 }
 
 message Announcement {
-  string id = 1;
-  string content = 2;
-  google.protobuf.Timestamp created_at = 3;
+  string announcement_id = 1;
+  storage.models.v2.AnnouncementType announcement_type = 2;
+  string title = 3;
+  string content = 4;
```

Comment:
Maybe an additional preview section ? Similar to a short description ?

Reply:
See above comment.